views/sections/forms/enroll.ejs

```diff
@@ -91,25 +91,16 @@
                     <div class="input-group col-lg-12 mb-4">
                         <div class="input-group-prepend">
                             <span class="px-1 border-md border-right-0 me-2">
-                                <img class="" src="/public/icon/social.png">
+                                <i class="bi bi-chat-left-text-fill fs-2" style="color: #ff4100;"></i>
                             </span>
                         </div>
 
-                        <div class="form-check pe-3">
-                            <input class="form-check-input" type="radio" name="groupChat" id="signal-radio" value="Signal" required>
-                            <label class="form-check-label" for="signal-radio">Signal</label>
-                        </div>
-
-                        <div class="form-check pe-3">
-                            <input class="form-check-input" type="radio" name="groupChat" id="telegram-radio" value="Telegram">
-                            <label class="form-check-label" for="telegram-radio">Telegram</label>
-                        </div>
-
-                        <div class="form-check">
-                            <input class="form-check-input" type="radio" name="groupChat" id="whatsapp-radio" value="WhatsApp">
-                            <label class="form-check-label" for="whatsapp-radio">WhatsApp</label>
-                        </div>
-
+                        <select class="form-control" id="class" name="groupChat" required>
+                            <option hidden value="">Size nereden ulaşalım? ↓</option>
+                            <option>Signal</option>
+                            <option>Telegram</option>
+                            <option>Whatsapp</option>
```

Comment:
This field must be strictly "WhatsApp" in order to not fail. Also, WhatsApp way coooler than Whatsapp :stuck_out_tongue_winking_eye: